_includes/parse-server/file-adapters.md

```diff
@@ -175,6 +175,94 @@ var api = new ParseServer({
 
 Don't forget to change **S3_ACCESS_KEY**, **S3_SECRET_KEY** and **S3_BUCKET** to their correct value.
 
+##### S3Adapter configuration for Digital Ocean Spaces
+
+Spaces is an S3 equivalent prodivided by Digital Ocean. It's use the same api as S3 so you can use it with the S3 Adapter.
+You just need to change the AWS Endpoint to point to your Spaces endpoint.
+
+```javascript
+...
+var S3Adapter = require('parse-server').S3Adapter;
+var AWS = require("aws-sdk");
+
+//Set Digital Ocean Spaces EndPoint
+const spacesEndpoint = new AWS.Endpoint(process.env.SPACES_ENDPOINT);
+//Define S3 options
+var s3Options = {
+  bucket: process.env.SPACES_BUCKET_NAME,
```

Comment:
Remove `process.env.` from all variables, this is specific solution we don't get get into in this example for the sake of compactness.